docs/source/glossary.rst

```diff
@@ -89,18 +101,28 @@ Glossary
         A mapping between :term:`grouping-schema <Grouping Schema>` common names, and their file locations on disk.
         File locations may be specified either as *relative* paths, with respect to <instrument.calibration.powder.grouping.home>, or as *absolute* paths.
 
-    HDF5
-        The most recent variant of Hierarchical Data Format (HDF).  A binary file format designed to scalably store and access large scientific data sets.
-
```

Comment:
This definition needs to remain

Reply:
This definition is retained. Just moved it elsewhere because it wasn't alphabetical.